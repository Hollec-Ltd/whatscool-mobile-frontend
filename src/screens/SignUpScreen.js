
import React from 'react';
import { View, Text, Button } from 'react-native';
import PropTypes from 'prop-types';

import { signUpScreenStyles } from '../styles';

const { screen } = signUpScreenStyles;
const SignUpScreen = (props) => (
  <View style={screen}>
    <Text>Some dummy text here! stands for SignUp page</Text>
    <Button
      title="Go back to Login page"
      onPress={() => {
        props.navigation.pop();
      }}
    />
  </View>
);

export default SignUpScreen;


SignUpScreen.propTypes = {
  navigation: PropTypes.object,
};
