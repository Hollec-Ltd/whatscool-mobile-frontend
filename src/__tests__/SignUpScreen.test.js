import React from 'react';
import { render } from 'react-native-testing-library';
import SignUpScreen from '../screens/SignUpScreen';

describe('SignUp Screen tests', () => {
  it('Should render SignUp component properly', () => {
    expect(<SignUpScreen />).toMatchSnapshot();
  });
});
