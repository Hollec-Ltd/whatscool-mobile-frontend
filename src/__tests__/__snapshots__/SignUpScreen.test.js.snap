// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp Screen tests Should render SignUp component properly 1`] = `<SignUpScreen />`;
